scripts/build_shapes.py

```diff
@@ -55,8 +55,8 @@ def download_GADM(country_code, update=False, out_logging=False):
 
     """
 
-    GADM_filename = f"gadm36_{two_2_three_digits_country(country_code)}"
-    GADM_url = f"https://biogeo.ucdavis.edu/data/gadm3.6/gpkg/{GADM_filename}_gpkg.zip"
+    GADM_filename = f"gadm41_{two_2_three_digits_country(country_code)}"
+    GADM_url = f"https://biogeo.ucdavis.edu/gadm/gadm4.1/gpkg/{GADM_filename}_gpkg.zip"
```

Comment:
The new file to download is a gpkg directly, before it was a .zip, there is the need to revise that.
Example: `https://geodata.ucdavis.edu/gadm/gadm4.1/gpkg/gadm41_ALB.gpkg`

Is short, we can skip the unzipping part